src/foam/nanos/menu/SequenceMenu.js

```diff
@@ -44,7 +44,7 @@ foam.CLASS({
       //
       // NOTE: it is by design for the first element of a stack to never be
```

Comment:
This note makes me think:
`first element of a stack` == (X.stack.pos == 0) ... so if that's not true anymore can you update the documentation, plz.